internal/k8s/secrets/store.go

```diff
@@ -111,14 +111,28 @@ type FakeSecretStore struct {
 }
 
 // NewFakeSecretsStore creates a new FakeSecretStore.
-func NewFakeSecretsStore(secrets map[string]*SecretReference) SecretStore {
+func NewFakeSecretsStore(secrets map[string]*SecretReference) *FakeSecretStore {
 	return &FakeSecretStore{
 		secrets: secrets,
 	}
 }
 
+// NewFakeSecretsStore creates a new empty FakeSecretStore.
```

Comment:
nit: incorrect function name in the comment
```suggestion
// NewEmptyFakeSecretsStore creates a new empty FakeSecretStore.
```